commcare_connect/templates/opportunity/opportunity_create.html

```diff
@@ -12,5 +12,28 @@
       {% crispy form %}
     </form>
 
+    <script>
+      window.onload = function () {
+        const applications = {{ applications|safe }};
```

Comment:
it's probably be better to use the `json_script` tag to convert the python dict to json safely : https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#std-templatefilter-json_script

Reply:
Thanks. I included this in the code. [05334bf](https://github.com/dimagi/commcare-connect/pull/37/commits/05334bf5e24d575d2c1a3e711ebd7b2edc5dcd65)